ui/app/styles/components/empty-state-component.scss

```diff
@@ -57,7 +57,7 @@
 }
 
 .empty-state-icon > .hs-icon,
-.empty-state-icon > .flight-icon {
+.empty-state-icon > .hds-icon {
```

Comment:
😬 `hds-icon` / `hs-icon`

Reply:
not ideal, but I imagine these overrides will be removed when the [Application State](https://helios.hashicorp.design/components/application-state) will be adopted